frontend/src/pages/signup-page.tsx

```diff
@@ -68,7 +67,7 @@ function Signup() {
     <div className="flex-grow cursor-default bg-white py-4 dark:bg-dark-card">
       <div className="m-4 mb-4 flex justify-center">
         <div className="flex w-full items-center justify-center">
-          <h2 className="w-3/4 pl-48 text-center text-lg font-bold text-black dark:text-dark-primary sm:text-xl">
+          <h2 className="text-center text-lg font-bold text-black dark:text-dark-primary sm:w-2/4 sm:pl-2 sm:text-xl lg:w-3/4 lg:pl-48">
             Sign up to WanderLust
```

Comment:
you mean to say `w-2/4 text-xl md:w-3/4 md:pl-48`? 

You are almost there, figure out why i said the above instead of `sm:w-2/4 sm:pl-2 sm:text-xl lg:w-3/4 lg:pl-48`

ps: We are focusing on mobile and devices with greater than mobile width screen . i.e md: and default should only be used, you can see the repo, no where we are using the `lg:`, you have solution here, please change it and try to understand why this change i suggested :)

Reply:
> you mean to say `w-2/4 text-xl md:w-3/4 md:pl-48`?
> 
> You are almost there, figure out why i said the above instead of `sm:w-2/4 sm:pl-2 sm:text-xl lg:w-3/4 lg:pl-48`
> 
> ps: We are focusing on mobile and devices with greater than mobile width screen . i.e md: and default should only be used, you can see the repo, no where we are using the `lg:`, you have solution here, please change it and try to understand why this change i suggested :)

ok now i understood by default it will be desktop and for mobile and screen small than desktop we should add sm,md for that right ?

Reply:
> ok now i understood by default it will be desktop and for mobile and screen small than desktop we should add sm,md for that right ?

Default is for mobile @kapilG0 in tailwind-css, in bootstrap it was what you are saying

Reply:
> > ok now i understood by default it will be desktop and for mobile and screen small than desktop we should add sm,md for that right ?
> 
> Default is for mobile @kapilG0 in tailwind-css, in bootstrap it was what you are saying

oh ok. thank you for clearing and helping me a lot in that . i have never use tailwind i will check some resource for that.

Reply:
No problem @kapilG0 , the hunger for learning, motivated me to help you :)

Reply:
> No problem @kapilG0 , the hunger for learning, motivated me to help you :)

thanks @krishnaacharyaa